test/passes/optimize-instructions_all-features.wast

```diff
@@ -4517,6 +4517,15 @@
       )
       (i32.const 0)
     ))
+    ;; (signed)x % 4 ? 1 : 0
+    (drop (if (result i32)
+      (i32.rem_s
+        (local.get $x)
+        (i32.const 4)
+      )
+      (i32.const 1)
+      (i32.const 0)
+    ))
   )
```

Comment:
oh, also, please add a testcase here for the min/max.

lgtm with that, sorry for not noticing before.

Reply:
Done!

Reply:
Great!